### ISSUE TITLE
scipy.stats.gaussian_kde raises `TypeError: Cannot read property 'apply' of undefined` 

### ISSUE THREAD

**Issue by ruoyu0088** [iodide-project/pyodide#615]
There is the code:

```python
%% py
from scipy import stats
a = np.random.randn(100)
kernel = scipy.stats.gaussian_kde(a)
```

and the error output:

```
TypeError: Cannot read property 'apply' of undefined
env.<computed> (http://localhost:8000/pyodide.asm.js:8:9126)
byn$fpcast-emu$_ddet_c_ (wasm-function[2106]:0x371818::)
_f2py_rout__flinalg_ddet_c (wasm-function[131]:0x13645::)
byn$fpcast-emu$_f2py_rout__flinalg_ddet_c (wasm-function[2107]:0x37182b::)
_fortran_call (wasm-function[148]:0x185be::)
byn$fpcast-emu$_fortran_call (wasm-function[2134]:0x371a78::)
wasm-function[840]:0x1338a6::
wasm-function[2752]:0x328894::
wasm-function[2747]:0x3211d2::
wasm-function[20579]:0x9835bc::
```

**Comment by Ralts1337** [comment]
Another scipy error here 
```
%% py 
import scipy.stats as stats
y1 = [1,2,3,4,5,6]
y2 = [2,3,4,5,6,7]
a,b = stats.ttest_ind(y1,y2)
[a,b]
```
however if y2 = [1,2,3,4,5,6] the output gives [0,1] correctly

error output from iodide:  basically the same as the first post
chrome says:
```
TypeError: Cannot read property 'apply' of undefined
env.<computed> (https://pyodide.cdn.iodide.io/pyodide.asm.js:8:9126)
_spmpar_ (wasm-function[635]:0x8db1a::)
_bratio_ (wasm-function[579]:0x7e4b4::)
_cumbet_ (wasm-function[595]:0x84cca::)
_cumt_ (wasm-function[605]:0x86004::)
_cdft_ (wasm-function[593]:0x84699::)
_cdft1_wrap (wasm-function[342]:0x49548::)
byn$fpcast-emu$_cdft1_wrap (wasm-function[1200]:0xed2bb::)
___pyx_f_5scipy_7special_7_ufuncs_loop_d_dd__As_dd_d (wasm-function[231]:0x3da48::)
byn$fpcast-emu$___pyx_f_5scipy_7special_7_ufuncs_loop_d_dd__As_dd_d (wasm-function[921]:0xec199::)
```

firefox says: 
```
ypeError: Module[prop] is undefined
get/env[prop] (https://pyodide.cdn.iodide.io/pyodide.asm.js:8:9106)
_spmpar_ (https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[635]:0x8db1a::)
_bratio_ (https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[579]:0x7e4b4::)
_cumbet_ (https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[595]:0x84cca::)
_cumt_ (https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[605]:0x86004::)
_cdft_ (https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[593]:0x84699::)
_cdft1_wrap (https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[342]:0x49548::)
byn$fpcast-emu$_cdft1_wrap (https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[1200]:0xed2bb::)
___pyx_f_5scipy_7special_7_ufuncs_loop_d_dd__As_dd_d (https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[231]:0x3da48::)
byn$fpcast-emu$___pyx_f_5scipy_7special_7_ufuncs_loop_d_dd__As_dd_d (https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[921]:0xec199::)
https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[985]:0x6fa61::
https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[978]:0x66efc::
https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[960]:0x5b071::
https://pyodide.cdn.iodide.io/pyodide.asm.js line 8 > WebAssembly.Module:wasm-function[2048]:0xcf2ac::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[840]:0x1338a6::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2752]:0x328894::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2747]:0x32110e::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[20579]:0x9835bc::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[838]:0x13319c::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[834]:0x13246a::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[833]:0x131fe6::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[850]:0x134ff5::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[876]:0x13693b::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[15040]:0x9707e7::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[845]:0x1346f0::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2747]:0x3217b6::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[20579]:0x9835bc::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2742]:0x315786::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[841]:0x133b3c::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2752]:0x328888::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2747]:0x3210fd::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[20579]:0x9835bc::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[838]:0x13319c::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[841]:0x133ab1::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2752]:0x328888::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2747]:0x32116c::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[20579]:0x9835bc::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[838]:0x13319c::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[841]:0x133ab1::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2752]:0x328888::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2747]:0x32116c::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[20579]:0x9835bc::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2742]:0x315786::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[841]:0x133b3c::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2752]:0x328888::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2747]:0x32110e::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[20579]:0x9835bc::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2742]:0x315786::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2741]:0x3146c2::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2689]:0x30fcec::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[15959]:0x973616::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[842]:0x133e74::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[843]:0x134479::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2752]:0x328752::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[2747]:0x32116c::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[20579]:0x9835bc::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[838]:0x13319c::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[834]:0x13246a::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[833]:0x131fe6::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[859]:0x135bf1::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[861]:0x135dd2::
https://pyodide.cdn.iodide.io/pyodide.asm.wasm:wasm-function[385]:0xfe2b8::
pyodide/Module.__runPython (https://pyodide.cdn.iodide.io/pyodide.asm.js:8:2033530)
_runpython_init_js/Module._runPythonInternal (https://pyodide.cdn.iodide.io/pyodide.asm.js:8:42942)
internal (https://pyodide.cdn.iodide.io/pyodide.asm.js:8:43507)
runInternal (https://pyodide.cdn.iodide.io/pyodide.asm.js:8:43868)
```

**Comment by rth** [comment]
Thanks for the report! Related to https://github.com/iodide-project/pyodide/issues/492

It's likely an issue with using Fortran compiled function (https://github.com/iodide-project/pyodide/issues/184) that currently sometime don't work well. The first example calls into flinalg via f2py, the second calls cdft1_wrap which is a wrapper around `scipy.special.cdflib` in Fortran.

The long term solution might be addressing https://github.com/iodide-project/pyodide/issues/476 and hoping that Fortan compilers for LLVM will get more mature then (and support WASM). Short term, I'm not really sure what we can do to improve this situation. Though if someone is willing to investigate that would be helpful.

**Comment by rth** [comment]
As far as I can tell this was also fixed by #1102 which will be included in the next release. The original example now works fine in the dev console. Please comment if you still experience issues.
